plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java

```diff
@@ -2165,6 +2166,22 @@ protected IStatus run(DBRProgressMonitor monitor) {
 
         transactionStatusUpdateJob = new TransactionStatusUpdateJob();
         transactionStatusUpdateJob.schedule();
+        partListener = new AbstractPartListener() {
```

Comment:
Looks a bit over-complicated.
Maybe we'd better check that editor is currently active in the TransactionStatusUpdateJob ?